api/cli/defaults.go

```diff
@@ -5,7 +5,7 @@ package cli
 const (
 	defaultBindAddress     = ":9000"
 	defaultDataDirectory   = "/data"
-	defaultAssetsDirectory = "."
+	defaultAssetsDirectory = "/public"
```

Comment:
This should not be changed, this would impact users which are not using the Docker image (tarballs setups) and it would not be compatible with the current Dockerfile.